problems/1.types+operators.js

```diff
@@ -2,16 +2,24 @@
 * @param {any} val - The input value
 * @returns {boolean} - Is val a number
 */
-function isNumber() {
-
+function isNumber(val) {
+  if(typeof(val) === "number") {
```

Comment:
Because the condition already evaluates to true or false, you can just return the condition.